app/Kconfig

```diff
@@ -89,6 +89,30 @@ config ZMK_BLE_PASSKEY_ENTRY
 	bool "Experimental: Requiring typing passkey from host to pair BLE connection"
 	default n
 
+config BT_GAP_AUTO_UPDATE_CONN_PARAMS
```

Comment:
Since it is the default I can go ahead and remove it but I liked to specify it in case they do decide to change it to N by default. I will go ahead and remove it though and we can just add it if it does change in the future.